pom.xml

```diff
@@ -0,0 +1,70 @@
+<project xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xmlns="http://maven.apache.org/POM/4.0.0"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
+
+    <modelVersion>4.0.0</modelVersion>
+    <groupId>jheappo</groupId>
+    <artifactId>jheappo</artifactId>
+    <packaging>jar</packaging>
+    <version>0.1</version>
+    <name>jheappo</name>
+    <description>JHeappo Heap Parser</description>
+
+    <properties>
+        <neo4j.version>3.3.5</neo4j.version>
+    </properties>
+
+    <build>
+        <plugins>
+            <plugin>
```

Comment:
versions and default config can go into a management section, I'll tidy that later though.